rust/headless-client/src/ipc_service.rs

```diff
@@ -529,6 +553,7 @@ impl<'a> Handler<'a> {
             Arc::new(tcp_socket_factory),
         )
         .map_err(|e| Error::PortalConnection(e.to_string()))?;
+        phoenix_span.finish();
```

Comment:
Other than resolving DNS, `connect` doesn't do anything. We will only connect once you start polling `PhoenixChannel`.

Reply:
I'll change the description, then. I am curious about how long we spend on synchronous DNS